### ISSUE TITLE
self user highlights no longer working

### ISSUE THREAD

**Issue by ProGoldfish** [Chatterino/chatterino2#3814]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

my own messages are no longer being highlighted in recent nightly update

### Screenshots

https://i.imgur.com/zXyu7Ov.png
works for my alt (prime badge) but not for the logged in account (artist badge)
https://i.imgur.com/55OV5kz.png

### OS and Chatterino Version

Chatterino Nightly 2.3.5 (commit 683701623 modified) built on with Qt 5.15.2, Windows SDK, MSVC 193231329 Running on Windows 10 Version 2009, kernel: 10.0.22000

**Comment by Felanbird** [comment]
Can replicate - side effect of 7ccf60111d5e8ba54c070fe09d926bdff5290da5

**Comment by Mm2PL** [comment]
```
[2022-06-7 11:54:47] #pajlada mm2pl: FeelsNotSureLady dankerino#124
[2022-06-7 11:54:48] #pajlada mm_sutilitybot: @Mm2PL https://github.com/Mm2PL/chatterino2/issues/124
[2022-06-7 11:55:00] #pajlada mm2pl: were self-highlights ever intended to work in c2?
[2022-06-7 11:55:02] #pajlada mm2pl: FeelsNotSureLady
[2022-06-7 11:55:07] #pajlada occluder: don't think so ?
[snip]
[2022-06-7 11:56:48] #pajlada pajlada: @Mm2PL, No
[2022-06-7 11:57:10] #pajlada pajlada: I know someone has asked for being able to add yourself as a user highlight
```

**Comment by Felanbird** [comment]
As a note, despite the above conversation I can replicate the functionality all the way back to 2.2.3, so while it may not have been intentionally added, it's been around long enough to now intentionally keep.

**Comment by LosFarmosCTL** [comment]
> so while it may not have been intentionally added, it's been around long enough to now intentionally keep.

I use it myself, and definitely know some other people that do as well. It's very useful for seeing if your messages went through in fast chats and just for a general overview of how many messages you have sent recently in the scrollbar.

I missed out on testing that PR or I would have noticed earlier.

**Comment by jammehcow** [comment]
#3708 might had something to do with this. We did have a discussion in pajlada's chat at the time where we agreed this shouldn't be default behaviour

**Comment by jammehcow** [comment]
Scratch that, most likely from the highlighting changes.

**Comment by Felanbird** [comment]
> #3708 might had something to do with this. We did have a discussion in pajlada's chat at the time where we agreed this shouldn't be default behaviour

> Scratch that, most likely from the highlighting changes.

I was able to replicate it working back 1 commit ( 6c38d3ecab7cc424e02fa7ad0f57a627bcaff06f ) before highlight refactor

**Comment by byaura** [comment]
Same, everyone that I know is using the old commit because of that, that's ridiculous

**Comment by Felanbird** [comment]
> that's ridiculous

Ridiculous would be deleting all prior version directly after this bug is introduced just to piss people off, this is just a bug, it happens.

**Comment by pajlada** [comment]
I believe this has been fixed by #3835 
If you could, please test this out and confirm its functionality

To test a PR build, you can browse to the linked PR, click the "Checks" tab, then select the "Build" job on the left side, then scroll down to find the build that fits your platform. Note that you need to be signed into GitHub to download from that page.
